msteams-platform/graph-api/meeting-transcripts/get-transcripts-api.md

```diff
@@ -0,0 +1,891 @@
+---
+title: Use Microsoft Graph to fetch transcripts for a Teams meeting
+description: Describes the process, scenarios, and APIs to fetch transcripts in a post-meeting scenario 
+ms.localizationpriority: medium
+ms.topic: concept
+keywords: teams Graph transcripts meeting organizer Azure rsc
+---
+# Get meeting transcripts using Graph APIs
+
+You can now configure your app to fetch Teams meeting transcripts in post-meeting scenario. Your app can use Graph REST APIs to access and fetch transcripts generated for a Teams meeting that has been scheduled beforehand.
+
+Here are some use cases for fetching meeting transcripts using Graph API:
+
+| Use case | How Transcript APIs help... |
+| --- | --- |
+| You need to obtain transcripts for capturing meaningful insights from multiple meetings across the Sales vertical. It's time-consuming and inefficient to keep track of all meetings, and to retrieve meeting notes manually. After the meeting is over, you'd need to examine conversations in all those meetings to obtain meaningful information. | If you configure your app to fetch meeting transcripts automatically, it gets the transcripts from all meetings relevant for your purpose. Your app can receive meeting notifications, and get the transcript when it's generated after the meeting. This data can then be used to gain: <br> • Aggregated insights and intelligence analysis <br> • New leads and highlights <br> • Meeting follow-ups and summaries |
```

Comment:
`You stand to lose useful valuable or suggestions.`  Might want to rephrase this?